### ISSUE TITLE
No connection possible with Blogger - 2020

### ISSUE THREAD

**Issue by undercovergypsy** [OpenLiveWriter/OpenLiveWriter#902]
I read the closed, "no connection with blogger" from 2015 but it is 2020 and man did walk on the moon even before 2015...Did people stop using OLW after 2015? Is anyone else getting this error from google? Please tell me that there is a work around???
![Annotation 2020-02-23 075835](https://user-images.githubusercontent.com/60517809/75117131-d9b2df80-5612-11ea-9542-feafd019181c.png)

**Comment by Matheuslaville** [comment]
Eu também infelizmente estou com o mesmo problema, e já reparei que nessa junção sempre no meio de ano ou no fim dar algum problema entre o Google e o Open Live Writer! Lamentável!! E o pior é que sempre sobra para o pessoal do Blogger como eu e vocês!

**Comment by undercovergypsy** [comment]
Yes! There are those of us left. And why not? It is still free and has great customization choices.

**Comment by Matheuslaville** [comment]
I agree with you!!

**Comment by GLComputing** [comment]
Same request ... Sad that this seems to have died

**Comment by RJCreationzzz** [comment]
> Same request ... Sad that this seems to have died

Login issue can be fixed please check this link: https://www.hackinguniversity.in/2020/04/fix-blogger-sign-in-error-open-live-writer.html

**Comment by RJCreationzzz** [comment]
> I read the closed, "no connection with blogger" from 2015 but it is 2020 and man did walk on the moon even before 2015...Did people stop using OLW after 2015? Is anyone else getting this error from google? Please tell me that there is a work around???
> ![Annotation 2020-02-23 075835](https://user-images.githubusercontent.com/60517809/75117131-d9b2df80-5612-11ea-9542-feafd019181c.png)

Login issue can be fixed please check: https://www.hackinguniversity.in/2020/04/fix-blogger-sign-in-error-open-live-writer.html

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
